source/sass/patterns/organisms/content-header.scss

```diff
@@ -95,6 +107,7 @@ $contentHeader-image-inline-padding: $contentHeader-image-block-padding !default
 
 .content-header__title {
   @include block-spacing($end: $baselinegrid-space-small);
+  overflow-y: hidden;
```

Comment:
Found that the title can be scrolled (a fraction). Not sure what this is the case, could well be something else wrong.

Reply:
Probably a `line-height` problem (https://elifesciences.slack.com/archives/CFZAFHK52/p1563464271007800?thread_ts=1563462858.003600&channel=CFZAFHK52&message_ts=1563464271.007800).

Reply:
Taken out the overflow in a52e266595e2f885a7c6efe59edf74d3bee4e5dd as this doesn't feel right. Too-long content will now overflow, but 🤷‍♂.